tools/ci_build/amd_hipify.py

```diff
@@ -131,12 +131,9 @@
                 'object_detection/roialign.h',
                 'object_detection/roialign_impl.cu',
                 'object_detection/roialign_impl.h',
-                'reduction/reduction_functions.cc',
                 'reduction/reduction_functions.cu',
```

Comment:
Could reduction_functions.cu be converted from corresponding CUDA file?

Reply:
Updated to reuse directly from CUDA.